README-PLAYBOOK.md

```diff
@@ -1,164 +1,164 @@
+| keyFile              | string  | ***REQUIRED*** when [Installation.format](#installation) is `PEM`. Specifies the file path and name for the private key PEM file (Example `/etc/ssl/certs/myKey.key`). *IGNORED* for other values of [Installation.format](#installation) |
+| location             | string |  ***REQUIRED*** when [Installation.format](#installation) is `CAPI`. Specifies the Windows CAPI store to place the installed certificate. Typically `"LocalMachine\My"` or `"CurrentUser\My"`. *IGNORED* if [Installation.format](#installation) is NOT `CAPI`. **NOTE:** if the location is not contained within `"`, the backslash `\` must be properly escaped (i.e. `LocalMachine\\My`). |
+| capiIsNonExportable  | boolean | *OPTIONAL* - When `true`, private key will be flagged as 'Non-Exportable' when stored in Windows CAPI store. *IGNORED* if [Installation.format](#installation) is NOT `CAPI`. Defaults to `false`. |
+
 
 ### Request
 
 | Field           | Type                   | Description |
-|-----------------|------------------------|-------------|
-| cadn            | string                 ||
-| chainOption     | string                 ||
-| csrOrigin       | string                 ||
-| customFields    | array of `CustomField` ||
-| dnsNames        | array of string        ||
-| emails          | array of string        ||
-| fetchPrivateKey | boolean                ||
-| friendlyName    | string                 ||
-| ips             | array of string        ||
-| issuerHint      | string                 ||
-| keyCurve        | string                 ||
-| keyLength       | integer                ||
-| keyPassword     | string                 ||
-| keyType         | string                 ||
-| location        | `Location`             ||
-| omitSans        | boolean                ||
-| origin          | string                 ||
-| subject         | `Subject`              ||
-| upns            | array of string        ||
-| uris            | array of string        ||
-| validDays       | string                 ||
-| zone            | string                 ||
-
-### CustomField
-
-| Field   | Type      | Description |
-|---------|-----------|-------------|
-| type    | string    ||
-| name    | string    ||
-| value   | string    ||
-
+|---|---|---|
+| chain           | string                 | *OPTIONAL* - Determines the ordering of certificates within the returned chain. Valid options are `root-first`, `root-last`, or `ignore`. Defaults to `root-last`. |
+| csr             | string                 | *OPTIONAL* - Specifies where the CSR and PrivateKey are generated: use `local` to generate the CSR and PrivateKey locally, or `service` to have the PrivateKey and CSR generated by the specified [Connection.platform](#connection). Defaults to `local`. |
+| fields          | array of strings)      | *OPTIONAL* - Sets the specified custom field on certificate object. Only valid when [Connection.platform](#connection) is `tpp`. |
```

Comment:
This is not an array of Strings, it is an array of `CustomField` object. Which I think is missing in the doc:

### CustomField

| Field   | Type      | Description |
|---------|-----------|-------------|
| name    | string    ||
| value   | string    ||